study-builder/fdahpStudyDesigner/.project

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should this have copyright header?

Reply:
This file is not required, so removed this file.